CMakeLists.txt

```diff
@@ -204,43 +201,29 @@ if (BUILD_DOCS)
   if (NOT SPHINX_FOUND)
     message(SEND_ERROR "Sphinx not found but BUILD_DOCS enabled")
   endif()
-  set(SPHINX_THEME "alabaster")
-  set(SPHINX_THEME_DIR)
-  set(SPHINX_BINARY_BUILD_DIR "${CMAKE_CURRENT_SOURCE_DIR}/docs/_build")
-  set(SPHINX_CACHE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/docs/_build/_doctrees")
-  set(SPHINX_HTML_DIR "${CMAKE_CURRENT_SOURCE_DIR}/docs/html/")
-
-  configure_file(
-    "${CMAKE_CURRENT_SOURCE_DIR}/docs/conf.py.in"
-    "${SPHINX_BINARY_BUILD_DIR}/conf.py"
-    @ONLY)
 
   file(GLOB SPHINX_DEPS
     "${CMAKE_CURRENT_SOURCE_DIR}/docs/*.rst"
     "${CMAKE_CURRENT_SOURCE_DIR}/docs/images/*.png"
-    "${CMAKE_CURRENT_SOURCE_DIR}/docs/conf.py.in"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*.lua"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/*.lua"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/*/*.lua"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/*/*/*.lua"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/*/*/*/*.lua"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/*/*/*/*/*.lua"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/*/*/*/*/*/*.lua"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/*/*/*/*/*/*/*.lua"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/*/*/*/*/*/*/*/*.lua"
-    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/*/*/*/*/*/*/*/*/*.lua"
+    "${CMAKE_CURRENT_SOURCE_DIR}/docs/styles/*"
+    "${CMAKE_CURRENT_SOURCE_DIR}/docs/conf.py"
+    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/about.txt"
+    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*/about.txt"
+  )
+  file(GLOB_RECURSE SPHINX_SCRIPT_DEPS scripts "${CMAKE_CURRENT_SOURCE_DIR}"
```

Comment:
Actually, I think this would be better:
```cmake
  file(GLOB_RECURSE SPHINX_SCRIPT_DEPS
    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*.lua"
    "${CMAKE_CURRENT_SOURCE_DIR}/scripts/*.rb"
  )
```